v2/dtos/responses/event.go

```diff
@@ -73,13 +73,13 @@ func NewMultiEventsResponse(requestId string, message string, statusCode int, ev
 	}
 }
 
-func NewUpdateEventPushedByChecksumResponseNoMessage(requestId string, statusCode int, checksum string) UpdateEventPushedByChecksumResponse {
-	return NewUpdateEventPushedByChecksumResponse(requestId, "", statusCode, checksum)
+func NewUpdateEventPushedByIdResponseNoMessage(requestId string, statusCode int, id string) UpdateEventPushedByIdResponse {
```

Comment:
please don't modify this func, because it will be removed in another PR

Reply:
Well, I have to modify this func, otherwise, I will have to add back `func NewUpdateEventPushedByChecksumResponse(requestId string, message string, statusCode int, checksum string)` , and then add back `NewUpdateEventPushedByChecksumResponse`, which is almost revert this entire PR.